sipservice/build.gradle

```diff
@@ -1,6 +1,7 @@
 apply plugin: 'com.android.library'
 apply plugin: "maven-publish"
 apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
```

Comment:
Just for a note -> 'kotlin-android-extensions' is deprecated in new version so we might have to remove it later. for current version its fine.

Reply:
Good catch! 

Resolved now, replaced with `kotlin-parcelize`